src/Vlingo.Actors/Plugin/Mailbox/AgronaMPSCArrayQueue/ManyToOneConcurrentArrayQueueDispatcher.cs

```diff
@@ -57,12 +59,15 @@ public void Run()
 
         public void Start()
```

Comment:
Please consider the two comments from the Run() method in RingBufferDispatcher.cs file in this file as well.